src/libraries/Common/src/Interop/Windows/Kernel32/Interop.Globalization.cs

```diff
@@ -85,7 +85,7 @@ internal static unsafe partial class Kernel32
                     int cchCount2,
                     bool bIgnoreCase);
 
-        [DllImport("kernel32.dll", EntryPoint = "FindStringOrdinal")]
+        [DllImport("kernel32.dll", EntryPoint = "FindStringOrdinal", SetLastError = true)]
```

Comment:
`SetLastError = true` is going to add an extra frame for the marshaling. What does the performance looks like with this change?

Reply:
Maybe it is good enough to just check the last error in debug builds.